### DISCUSSION TITLE
Skip the container update if its a template

### DISCUSSION THREAD

**Discussion by Gauvino** [tteck/Proxmox#825] [Ideas]
When updating the container make a condition for template to skip it

You can know if its a template or not via pct config <ctid> = template: 1

It would be very cool cause i think im not the only one whit this problem

https://raw.githubusercontent.com/tteck/Proxmox/main/misc/update-lxcs.sh

**Comment by tteck** [comment]
~~It does that now, templates are not running. It asks to skip stopped containers.~~

**Comment by tteck** [comment]
The script has been updated to skip all templates, allowing for the starting, updating, and shutting down of containers to be resumed automatically.
If you have an additional container that you would like to exclude, you can add the CTID at the end of the shell command like this:
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs.sh)" -s 103
```
Replace `103` with the CTID of the container that you want to exclude. This will skip updating that container.
